JenkinsFile

```
@@ -0,0 +1,10 @@
+/*
+ See the documentation for more options:
+ https://github.com/jenkins-infra/pipeline-library/
+*/
+buildPlugin(
+  useContainerAgent: true, // Set to `false` if you need to use Docker for containerized tests
+  configurations: [
+    [platform: 'linux', jdk: 17],
```

I don't think that you want to drop Java 11 support yet.
```suggestion
    [platform: 'linux', jdk: 11],
```